Re-modularize the Monitoring Hosts section (#2555)
theforeman/foreman-documentation: guides/common/modules/proc_remove-hosts-from-red-hat-insights-inventory-and-satellite-simultaneously.adoc

[id="remove-hosts-from-red-hat-insights-inventory-and-satellite-simultaneously_{context}"]

> = Remove Hosts from Red Hat Insights Inventory and Satellite simultaneously

Reviewer: ```suggestion
= Configuring Automatic Removal of Hosts from the Insights Inventory
```
I'm thinking about a simpler, clearer, and more precise title here. How about this?

Author: Yes, better. Thanks :+1:
